code/modules/busy_space/organizations.dm

```diff
@@ -463,27 +635,70 @@
+		/datum/lore/system/neon_light = 1,
 		)
 
 /datum/lore/organization/tsc/morpheus
-	name = "Morpheus Cyberkinetics"
-	short_name = "Morpheus"
-	acronym = "MC"
+	name = "Morpheus Cyberkinetics - Sol Branch"
+	short_name = "Morpheus Sol"
+	acronym = "MC-S"
 	desc = "The only large corporation run by positronic intelligences, Morpheus caters almost exclusively to their sensibilities \
-	and needs. A product of the synthetic colony of Shelf, Morpheus eschews traditional advertising to keep their prices low and \
+	and needs. Originally product of the synthetic colony of Shelf, Morpheus eschews traditional advertising to keep their prices low and \
 	relied on word of mouth among positronics to reach their current economic dominance. Morpheus in exchange lobbies heavily for \
 	positronic rights, sponsors positronics through their Jans-Fhriede test, and tends to other positronic concerns to earn them \
-	the good-will of the positronics, and the ire of those who wish to exploit them."
+	the good-will of the positronics, and the ire of those who wish to exploit them. Morpheus Sol is legally distinct from its \
+	Shelfican counterpart, though both are notionally headed by the same board of directors."
 	history = ""
 	work = "cybernetics manufacturer"
-	headquarters = "Shelf"
+	headquarters = "Sophia, El" //ancient discord lore but also just a really reasonable extrapolation from everything
 	motto = ""
+	legit = 75 // hey remember that time they blew up a star. and then everything sucked for everyone.
+
+	missions = list(
+		new /datum/lore/mission/prebuilt/freight("MFV"),
+		new /datum/lore/mission/prebuilt/transport("MTV"),
+		new /datum/lore/mission/prebuilt/scientific("MSV"), // this is the MSV We Didn't Do It! on a data collection flight to an anomaly in whythe
+		new /datum/lore/mission/prebuilt/defense("MDV"), //ive been giving defense fleets to every "major" TSC but morph's narrow focus and Obvious Sleeziness means they might not even want one?
+		)
 
-	ship_prefixes = list("MTV" = "freight")
-	// Culture names, because Anewbe told me so.
+	serviced = list( //pretty flat weighting structure -- a lot of posi-majority systems are on the other end of Sol so it kinda works out that way
+		/datum/lore/system/vir = 15,
+		/datum/lore/system/el = 15,
+		/datum/lore/system/sol = 10,
+		/datum/lore/system/gavel = 10,
+		/datum/lore/system/oasis = 10, //somehow i imagine they do not really operate openly in Saint Columbia anymore
+		/datum/lore/system/kess_gendar = 5,
+		/datum/lore/system/tau_ceti = 5,
+		/datum/lore/system/terminus = 10,
+		/datum/lore/system/raphael = 10,
+		/datum/lore/system/relan = 10,
+		/datum/lore/system/exalts_light = 5,
+		/datum/lore/system/new_ohio = 5,
+		/datum/lore/system/new_seoul = 5,
+		/datum/lore/system/zhu_que = 5, //apparently they have a voter farm??
+		/datum/lore/system/love = 2,
+		/datum/lore/system/neon_light = 2,
+		/datum/lore/system/alpha_centauri = 2,
+		/datum/lore/system/whythe = 1, //oh god
```

Comment:
```suggestion
		/datum/lore/system/whythe = 1 //oh god
```
AFAIK the wiki says Whythe is entirely too dangerous to go to? Should this be possible?

Reply:
it's "uninhabitable" and full of skathari but extremely rare missions to it should be fine IMO

Reply:
o7